README.md

```diff
@@ -39,4 +40,60 @@ chain-adapter:
     - rmq
   volumes:
     - ../your change adapter storage/:/deploy/chain-adapter
-  ```
\ No newline at end of file
+```
+  
+## How to use
+`com.d3.chainadapter.client.ReliableIrohaChainListener` is the class that is used as a client for the service. The class may be obtained via JitPack:
+
+```groovy
+compile "com.github.d3ledger.chain-adapter:chain-adapter-client:$chain_adapter_client_version"
+``` 
+Typical workflow looks like the following:
+
+1) First, you must create an instance of `ReliableIrohaChainListener` object. 
+2) Then you have to call the `getBlockObservable()` function that returns `Observable<Pair<BlockOuterClass.Block, () -> Unit>>` wrapped by `Result`(see [github.com/kittinunf/Result](https://github.com/kittinunf/Result)). The returned object may be used to register multiple subscribers.
+The first component of `Pair` is an Iroha block itself. The second component is a function that must be called to acknowledge Iroha block delivery. 
+The function won't make any effect if the "auto acknowledgment" mode is on. If the "auto acknowledgment" mode is off, then EVERY block must be acknowledged manually.  
```

Comment:
have an effect. Unless the mode is on? Not very easy to understand the point. Should it be on or off?

Reply:
Unfortunately, I see no clarity issues.